### ISSUE TITLE
Active tab group sent behind

### ISSUE THREAD

**Issue by bobweasel** [projectdelphai/panorama-tab-groups#241]
In tab group view, when clicking on a tab group, the group goes behind all other groups. This is annoying when the groups overlap each other and clicking on the group you want causes other groups to be above your group, hindering the access to the tabs.

**Comment by vaxul** [comment]
This is true. In this moment you could try the autoplacement within the freeform view or give the popup view a try. :)
